TimberAPI/EntityLinkerSystem/EntityLinkerConfigurator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bindito.Core;
+using TimberbornAPI.EntityActionSystem;
+
+namespace TimberbornAPI.EntityLinkerSystem
+{
+    public class EntityLinkerConfigurator : IConfigurator
+    {
+        public void Configure(IContainerDefinition containerDefinition)
+        {
+            containerDefinition.MultiBind<IEntityAction>().To<EntityActions>().AsSingleton();
```

Comment:
Maybe it's better for the API to use the decorator  if possible. Since it doesn't apply any special actions except adding the new monobehaviour to all.